library/X509/Web/Widget/EmptyList.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+// Icinga Web 2 X.509 Module | (c) 2023 Icinga GmbH | GPLv2
+
+namespace Icinga\Module\X509\Web\Widget;
+
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\Text;
+
+class EmptyList extends BaseHtmlElement
```

Comment:
We call that `empty-state` everywhere else, e.g. https://github.com/Icinga/icingadb-web/blob/master/library/Icingadb/Widget/EmptyState.php